src/OrchardCore.Modules/OrchardCore.Settings/Controllers/AdminController.cs

```diff
@@ -73,7 +74,13 @@ public async Task<IActionResult> IndexPost(string groupId)
             {
                 await _siteService.UpdateSiteSettingsAsync(site);
 
-                _notifier.Success(H["Site settings updated successfully."]);
+                var culture = site.Properties
```

Comment:
Please add a comment to explain why we create a new scope with a custom culture.
Something like "We create a transient scope with the newly selected culture to create a notification that will use it instead of the previous culture". In both places.